languages/en-US/administration/configuration.json

```
@@ -2,34 +2,34 @@
     "DESCRIPTION": "Shows the server configuration!",
     "USAGE": "{{prefix}}configuration",
     "EXAMPLES": "{{prefix}}configuration",
-    "PREFIX_TITLE": "Server prefix",
-    "IGNORED_CHANNELS_TITLE": "Ignored channel(s)",
+    "PREFIX_TITLE": "\n- Server prefix:",
+    "IGNORED_CHANNELS_TITLE": "- Ignored channel(s):",
```

Sans le tiret c'est plus propre visuellement 

```suggestion
    "PREFIX_TITLE": "\nServer prefix:",
    "IGNORED_CHANNELS_TITLE": "Ignored channel(s):",
```